### DISCUSSION TITLE
Why doesn't nuclei display anything on the screen while scanning?

### DISCUSSION THREAD

**Discussion by shadihh9** [projectdiscovery/nuclei#907] [Q&A] [answered]
when i scan the URL I don't anything nothings
![nm](https://user-images.githubusercontent.com/22076930/128317045-989e8dca-bfd6-445a-8158-87d2e8762b18.png)

**Comment by ehsandeep** [comment, marked as answer]
@shadihh9 Yes, that's the expected default screen, you will only see something if nuclei find results, if you want to see scan status, you can use flags like `-stats`
